Update content for reflecting new naming conventions (#97)
greenbone/docs: src/faq.md

For a comprehensive background see {doc}`History of OpenVAS <history>`.

> ## Where can I ask question and get support?

Reviewer: ```suggestion
## Where can I ask questions and get support?
```

> Support is only provided for the Greenbone Enterprise Appliance products.

Reviewer: ```suggestion
Support is only provided for the Greenbone Enterprise product line.
```

> Besides that the Greenbone Community can be reached at [https://community.greenbone.net][community forum]

Reviewer: ```suggestion
Besides that, the Greenbone Community can be reached at [https://community.greenbone.net][community forum]
```

> to ask question. In this forum several Greenbone developers trying to help on a

Reviewer: ```suggestion
to ask question. In this forum, several Greenbone developers trying to help on a
```

voluntary basis.

## Can you help with my issue on Kali, Cent OS, XYZ distribution?

[https://community.greenbone.net/t/openvas-showing-503-error/5465/4](https://community.greenbone.net/t/openvas-showing-503-error/5465/4)

Greenbone doesn't provide any packages for any Linux distribution besides the
commercial Greenbone OS. If you have installed the Greenbone Community Edition
software from your distribution like Kali Linux, an external package repository

> or even some random docker image Greenbone wasn't involved in providing this

Reviewer: ```suggestion
or even some random docker image, Greenbone was not involved in providing this
```

installation method to you.

The development of the Greenbone Community Edition and packaging for a
distribution or container image is completely independent. Therefore our
software provided from these sources may be heavily adjusted, outdated or even

> completely broken. If you have issues with the software please contact the

Reviewer: ```suggestion
completely broken. If you have issues with the software, please contact the
```

provider of the packages first and create a ticket at their issue tracker. How

> to contact the provider really depends and varies.

Reviewer: ```suggestion
to contact the provider depends and varies.
```

We are **not** able to offer any help on updating packages from **any** external
source like Kali, Cent OS, Docker Image, .... because we are not aware of their
specific needs and changes!

## Please help me installing OpenVAS and please advise on recommended platform OS, too.

This is very similar to [Can you help with my issue on Kali, Cent OS, XYZ distribution](#can-you-help-with-my-issue-on-kali-cent-os-xyz-distribution).
Greenbone can't provide installation docs for the many available Linux
distributions available. Setups, configurations, build systems, packaging tools,
available software, best practices, ... diverge to much to offer *official*
packages, scripts or anything else from Greenbone. We offer this Greenbone Community

> documentation and the [community forum] for discussing setup problems. Therefore

Reviewer: ```suggestion
documentation and the [community forum] for discussing setup problems. Therefore,
```

Greenbone also doesn't recommend any distribution nor do we prefer one

> distribution over the other. Most of the time we are even not aware which

Reviewer: ```suggestion
distribution over the other. Most of the time, we are even not aware which
```

distribution ships which version of our software.

## Which release contains which component? GOS version vs. GVM version?

It was often confusing to find out which software component of the
{term}`Greenbone Community Edition` belongs to which GVM release.

> Additionally the {term}`Greenbone OS <GOS>`

Reviewer: ```suggestion
Additionally, the {term}`Greenbone OS <GOS>`
```

used in the [Greenbone Enterprise TRIAL Virtual Machine](https://www.greenbone.net/en/testnow/)

> had a different versioning scheme then GVM. We are aware of this and therefore

Reviewer: ```suggestion
had a different versioning scheme than GVM. We are aware of this and therefore
```

> with the 20.08 release we changed our versioning scheme to

Reviewer: ```suggestion
with the 20.08 release, we changed our versioning scheme to
```

For a comprehensive background see {doc}`History of OpenVAS <history>`.

## Where can I ask questions and get support?

Support is only provided for the Greenbone Enterprise product line.

> Besides that, the Greenbone Community can be reached at [https://community.greenbone.net][community forum]

Reviewer 2: ```suggestion
Besides that, the Greenbone Community can be reached at the [https://community.greenbone.net][community forum]
```

project and not to get instant advises from the developers or professional
support to fix your current issue.

> If you are using a self-compiled version of our software stack not build from this

Reviewer 2: ```suggestion
If you are using a self-compiled version of the Greenbone software stack not build from this
```
